### DISCUSSION TITLE
Prácticas

### DISCUSSION THREAD

**Discussion by GerardoSolanoo** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#39] [Q&A]
Buenas tardes, tengo bastantes dudas a la hora de las prácticas puesto que no tengo idea si ya se aclaro el como llenarlas por que ya se ha puesto una fecha limite.

**Comment by miguelapol** [comment]
Que yo sepa compañero todavía no sube nada de sobre las prácticas tal vez el lunes nos den más información.

**Reply by CharlyTeddyEddy** [reply]
Las subieron hace 2 días en la sección del Intro!

**Reply by dnovellas** [reply]
Ya están arriba desde hace dos días. Este es el link:

[https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/21#discussion-3887473](Practicas)
[Practicas](https://github.com/LaunchX-InnovaccionVirtual/FrontEnd-Mision/discussions/21#discussion-3887473)

Saludos.

**Reply by YairRuizMx** [reply]
Comentó el buen commander que iba a subirnos un vídeo explicando cómo hacer las prácticas. D:

**Reply by OntiverosLuis** [reply]
Pero aun  no lo ha subido o ha explicado algo???...
No me gustaría atrasarme y a decir vrdd estoy algo nervioso jeje 
Saludos compañeros...

**Reply by RamsesCruz11** [reply]
Aun no sube el video luis, tu tranquilo, con que ayas leido la informacion y visto los lives ya sea grabados vas muy bien

**Comment by BriaAlba** [comment]
Estoy igual se subieron hace 2 días pero no han dicho nada mas que se entregaran el viernes.
